### PR TITLE
Added S1AreaFractionTop

### PR DESCRIPTION
**Pull request by darrylmasson**
Adds version 0 of the s1 area fraction top cut. The map is a separate json file that at some point should make its way into pax, but for now Jelle said to push it into lax.

### CODE REVIEW

**Review by tunnell — COMMENTED** [review]
This really needs to make its way into pax... can you do that too?  This should be thought of as temporary.

File: lax/lichens/sciencerun0.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # -*- coding: utf-8 -*-
 import numpy as np
+from scipy.stats import binom_test
 from pax import units, configuration
+from pax.InterpolatingMap import InterpolatingMap
```
Comment (tunnell) [inline lax/lichens/sciencerun0.py:5]: Is there a way to *not* use the pax interpolating map?  Scipy interpolate or something?  It's another (big) dependency.

**Review by tunnell — CHANGES_REQUESTED** [review]
You need to add scipy to setup.py.

File: lax/lichens/sciencerun0.py (hunk 1 above)
Comment (darrylmasson) [reply lax/lichens/sciencerun0.py:5]: Scipy doesn't have a multidimension map that both interpolates and extrapolates. In principle we could reimplement the pax map here and strip out the logging features, which should keep the extra dependencies to a minimum until the p-value is computed further up the processing chain.

File: lax/lichens/sciencerun0.py [hunk 2]
```diff
@@ -249,3 +251,35 @@ def _process(self, df):
...
+class S1AreaFractionTop(RangeLichen):
+    '''S1 area fraction top cut
+    
+    Uses scipy.stats.binom_test to compute a p-value based on the
+    observed number of s1 photons in the top array, given the expected
+    probability that a photon at the event's (x,y,z) makes it to the top array.
+    
+    Uses a 3D map generated with Kr83m 32 keV line
+    
+    note: xenon:xenon1t:darryl:xe1t_s1_aft_map
```
Comment (pdeperio) [inline lax/lichens/sciencerun0.py]: put actual link

**Review by tunnell — APPROVED** [review]

**Pull request merged**